ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -32,12 +37,15 @@ tag_specification:
   namespace: ocp
 test_binary_build_commands: |
   make build-openshift-ci-test-bin
+  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
+  go mod vendor
+  mkdir /.cache && chmod 775 -R /.cache && chmod 775 -R $GOPATH
 tests:
 - artifact_dir: /tmp/artifacts
   as: lint
-  commands: export GOLANGCI_LINT_CACHE=/tmp/.cache make lint
+  commands: export GOLANGCI_LINT_CACHE=/tmp/.cache && export GOPROXY=https://proxy.golang.org && make lint
```

Comment:
I think this code should be devided to another PR.

Reply:
I can see why it could be done in another PR but that isn't too important to me.